### PR TITLE
Homeatic signalgen

### PR DESCRIPTION
**Pull request by siom79**
**Description:**

Describes the new component ``notify.homematic_signalgen`` for the homematic signal generator.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16973

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Docs look great! Thanks! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by tjorim — CHANGES_REQUESTED** [review]
What is that `csv.markdown` doing in this PR?

File: source/_components/notify.homematic.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  type: integer
+param:
+  description: An additional parameter for the Homematic device. 
+  required: true
+  type: string
+value:
+  description: This is the value that is set on the device. Its device specific.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (tjorim) [inline source/_components/notify.homematic.markdown:53]: In the component code is also an optional `ATTR_INTERFACE`, is it missing here?

**Review by tjorim — CHANGES_REQUESTED** [review]

File: source/_components/notify.homematic.markdown (hunk 1 above)
Comment (siom79) [reply source/_components/notify.homematic.markdown:53]: @tjorim Yes, you are right. The ``interface`` parameter was missing. I have added it now and removed the ``csv.markdown``.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @siom79!

**Pull request merged**